### PR TITLE
Add HTTP Load Tests

### PR DESCRIPTION
**Pull request by ayeshLK**
Fixes [#2072](https://github.com/ballerina-platform/ballerina-standard-library/issues/2072)

## Examples
N/A

## Checklist
- [X] Linked to an issue
- [ ] ~Updated the changelog~
- [ ] ~Added tests~

### CODE REVIEW

File: load-tests/snowpeak/src/Colud.toml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[container.image]
```
Comment (dilanSachi) [inline load-tests/snowpeak/src/Colud.toml:1]: Cloud.toml

**Review by dilanSachi — APPROVED** [review]
LGTM

File: load-tests/snowpeak/src/Colud.toml (hunk 1 above)
Comment (ayeshLK) [reply load-tests/snowpeak/src/Colud.toml]: Fixed with [3441b7781a39e5ff139c21c49b023dda2ad46efc](https://github.com/ballerina-platform/module-ballerina-http/pull/791/commits/3441b7781a39e5ff139c21c49b023dda2ad46efc)

**Comment by shafreenAnfar** [comment]
How long does this test runs for ?

**Comment by shafreenAnfar** [comment]
Is the error rate is 0% ?

**Comment by ayeshLK** [comment]
> How long does this test runs for ?
1 hour

> Is the error rate is 0% ?
Yes

**Review by shafreenAnfar — APPROVED** [review]
LGTM

**Pull request merged**